src/app/component/Form/Form.component.js

```diff
@@ -39,7 +40,7 @@ class Form extends Component {
             if (child && typeof child === 'object' && child.type && child.props) {
                 const { type: { name }, props, props: { children } } = child;
 
-                if (name === Field.prototype.constructor.name) {
+                if (name === Field.prototype.constructor.name || name === Select.prototype.constructor.name) {
```

Comment:
Is this needed? Seems like you moved select into Field.

Reply:
Resolved.